src/styles.css

```
@@ -497,6 +497,16 @@
 
 	.workspace-document-prose {
+		/*
+		 * A readable measure, held by the container's own padding rather than a
+		 * max-width on each block. Blocks set margin shorthand (widgets, tables,
+		 * rules, block math all do), which would reset any inline auto-margin and
+		 * strand them left of the text. Padding cannot be overridden that way, and
+		 * the editable element stays full width so a click anywhere still focuses
+		 * the editor.
+		 */
+		--workspace-document-measure: 80ch;
+		padding-inline: max(1rem, calc((100% - var(--workspace-document-measure)) / 2));
```

Stylelint (17.14.1)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)